If we keep comm.diff, this patch is necessary to build apparmor as a
module.
---
 fs/exec.c |    2 ++
 1 file changed, 2 insertions(+)

Index: b/fs/exec.c
===================================================================
--- a/fs/exec.c
+++ b/fs/exec.c
@@ -807,6 +807,8 @@ void get_task_comm(char *buf, struct tas
 	task_unlock(tsk);
 }
 
+EXPORT_SYMBOL_GPL(get_task_comm);
+
 void set_task_comm(struct task_struct *tsk, char *buf)
 {
 	task_lock(tsk);
